static/docs/commands-reference/checkout.md

```diff
@@ -51,6 +51,10 @@ The output of `dvc checkout` does not list which data files were restored. It
 does report removed files and files that DVC was unable to restore due to it
 missing from the cache.
 
+This command will fail to checkout files that are missing from the cache. In
+such a case, `dvc checkout` prints a warning message. Any files that can be
+checked out without error will be restored.
```

Comment:
`any files that are found in cache` instead of an error. It's not an error usually - it's a warning like you mentioned above.